dotnet/src/Connectors/Connectors.UnitTests/OpenAI/KernelConfigTests.cs

```diff
@@ -0,0 +1,307 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Linq;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel;
+using Moq;
+using Xunit;
+
+namespace SemanticKernel.Connectors.UnitTests.OpenAI;
+
+/// <summary>
+/// Unit tests of <see cref="KernelConfig"/>.
+/// </summary>
+public class KernelConfigTests
```

Comment:
I would name it `KernelConfigOpenAIExtensionsTests` to explicitly specify that this class contains tests for OpenAI extension methods only. If I will add `KernelConfigHuggingFaceExtensions` I would like to keep tests for this class separately, but in this case I won't be able to name it `KernelConfigTests` twice. And it would be nice to avoid `partial` 😃